portal/src/main/webapp/WEB-INF/jsp/enrichments_tab.jsp

```diff
@@ -1,134 +1,31 @@
-<script type="text/javascript" src="js/src/enrichments/main.js?<%=GlobalProperties.getAppVersion()%>"></script>
-<script type="text/javascript" src="js/src/enrichments/data.js?<%=GlobalProperties.getAppVersion()%>"></script>
-<script type="text/javascript" src="js/src/enrichments/view.js?<%=GlobalProperties.getAppVersion()%>"></script>
-<script type="text/javascript" src="js/src/enrichments/util.js?<%=GlobalProperties.getAppVersion()%>"></script>
-<script type="text/javascript" src="js/src/enrichments/plots.js?<%=GlobalProperties.getAppVersion()%>"></script>
-<script type="text/javascript" src="js/src/enrichments/boilerplate.js?<%=GlobalProperties.getAppVersion()%>"></script>
-<script type="text/javascript" src="js/src/enrichments/stacked_histogram.js?<%=GlobalProperties.getAppVersion()%>"></script>
-<script type="text/javascript" src="js/lib/plotly.min.js?<%=GlobalProperties.getAppVersion()%>"></script>
-<script type="text/javascript" src="js/src/enrichments/ScatterPlotly.js?<%=GlobalProperties.getAppVersion()%>"></script>
-<script type="text/javascript" src="js/src/enrichments/MiniOnco.js?<%=GlobalProperties.getAppVersion()%>"></script>
-<script type="text/javascript" src="js/src/enrichments/volcano_plot_view.js?<%=GlobalProperties.getAppVersion()%>"></script>
-<link href="css/stacked_histogram.css?<%=GlobalProperties.getAppVersion()%>" type="text/css" rel="stylesheet" />
-
-<div class="section" id="enrichementTabDiv">
-    <div id="enrichments-tab-info-box" style="padding: 10px;margin-top: -20px;"></div>
-    <div id="enrichments-tab-tabs" class="enrichments-tab-tabs" style="margin-top:5px;">
-        <ul id='enrichments-tab-tabs-list'></ul>
-        <div id='enrichments-tab-tabs-content'></div>
-    </div>
+<div class="cbioportal-frontend">
```

Comment:
I think you have this class in the renderer function.  Probably not a good idea to nest them.